### ISSUE TITLE
Barcodes Not Appearing in Universe

### ISSUE THREAD

**Issue by LitsaWoo** [ArctosDB/arctos#8245]
I am trying to scan some barcodes into positions. @campmlc helped me figure things out a couple weeks ago and we were able to make it work. A student filled one box with no problems and was working on a second. She reloaded the page and everything she had scanned into it deleted itself. Now the containers say they won't fit into the parent even though all of the sizes are correct (including the dimensions for the positions).

When I search a barcode it doesn't show properly either (picture below). I tried to re-bulkload the barcodes but it did not work. Can't figure out what the issue is, especially when these were working with no problem beforehand

![Barcode issue](https://github.com/user-attachments/assets/02a0a153-d2d4-452c-8167-2129e583691b)

**Comment by dustymc** [comment]
The tree is broken because the container is in a container which doesn't exist. What I can see suggests you edited that with autoload_container_edits. Do you have that data by any chance?

**Comment by happiah-madson** [comment]
@dustymc it looks like these are all of the objects from a single freezer box. We have the positioned freezer box, so I'll move them over. I'm guessing they were just left behind accidentally during migration.

**Comment by happiah-madson** [comment]
Okay, @dustymc I lied, it looks like the box is created, but the positions are not? Given https://arctos.database.museum/containerPositions.cfm?container_id=22937142 do I just need to click "create all new positions"? The fields in "edit container" for this freezer box are identical to other freezer boxes that contain cryovials.

**Comment by dustymc** [comment]
>but the positions are not

From what little info I have, I think this involves deleting occupied positions. It's probably still possible to try and I'd still like to find that, but there's a proper key now so it shouldn't actually happen.

**Comment by happiah-madson** [comment]
I think I'm confused. When I look at the freezer box that our floating samples should be in (https://arctos.database.museum/findContainer.cfm?container_id=22937142) it says that there are no records. If I click on positions, I get https://arctos.database.museum/containerPositions.cfm?container_id=22937142. If I click on empty positions, I get https://arctos.database.museum/findEmptyFBP.cfm?container_id=22937142.

I guess I'm not sure what I should do next, save for create all new positions in the box.

**Comment by dustymc** [comment]
>create all new positions in the box

Correct, and then put the orphaned whatever it is from my CSV above into them - at least I can't see any better way to recover this.

(Mostly irrelevant now, my working theory is that someone's done all that then found a way to delete the positions.)

**Comment by happiah-madson** [comment]
Ach, okay, I see what you're saying re: cause. Okay, great, I shall proceed as discussed!
